.svgo.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
             focusable: false
           },
           {
-            role: 'presentation'
+            'aria-hidden': true
```

Comment:
This clears out SVG errors seen in Storybook's a11y tab and applies the correct accessibility semantics for visually decorative Garden icons. See https://css-tricks.com/accessible-svgs/#example-2-standalone-icon-decorative for details.